### PR TITLE
SideNav bug fix

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #:*

Fixed small mobile issue, sidebar showing before login and made side bar sticky.

### CODE REVIEW

File: met-web/src/components/layout/SideNav/SideNav.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        <>
+            {isMediumScreen ? (
+                <Box sx={{ paddingTop: '10vh', height: '100vh', width: 240 }}>
+                    <DrawerBox isMediumScreen={isMediumScreen} navigate={navigate} />
+                </Box>
+            ) : (
+                <Drawer
+                    sx={{
+                        width: '15%',
+                        background: '#003366',
```
Comment (tiagograf-AOT) [inline met-web/src/components/layout/SideNav/SideNav.tsx:57]: Let’s have some sort of constants/themes for colour codes like this

**Review by tiagograf-AOT — APPROVED** [review]
We need to look into how to move away from hardcoded values for styling properties

File: met-web/src/components/layout/SideNav/SideNav.tsx (hunk 1 above)
Comment (djnunez-aot) [reply met-web/src/components/layout/SideNav/SideNav.tsx:57]: Sure thing ill set something up

**Review by jadmsaadaot — DISMISSED** [review]
Cool idea to keep the background color in one place

**Review by jadmsaadaot — APPROVED** [review]

**Review by jadmsaadaot — APPROVED** [review]

**Review by tiagograf-AOT — APPROVED** [review]

**Pull request merged**